mdal/api/mdal.h

```diff
@@ -273,6 +273,16 @@ MDAL_EXPORT MDAL_MeshH MDAL_CreateMesh( MDAL_DriverH driver );
  */
 MDAL_EXPORT void MDAL_SaveMesh( MDAL_MeshH mesh, const char *meshFile, const char *driver );
 
+/**
+ * Saves mesh (only mesh structure) on a file with an uri. On error see MDAL_LastStatus for error type.
+ *
+ * uri has form <DriverName>:"<MeshFilePath>"[:<SpecificMeshName>]
+ * examples: Ugrid:"mesh.nc":0, Ugrid:"mesh.nc":mesh2d, Ugrid:"mesh.nc"
+ *
```

Comment:
- is `SpecificMeshName` used in any driver?

Reply:
not for now..., but UGRID has "mesh1D","mesh2D". Later, if UGRID support writing 1D, we will need that